src/scss/initial/_links.scss

```diff
@@ -21,14 +21,6 @@
 /// Text underlines are also applied,
 /// but the target depends on the presence or absence of icons,
 /// in order to avoid awkward-looking icon underlines.
-/// @example njk
```

Comment:
@mirisuzanne This was breaking because our macros use filters added via Eleventy. We need to use the same Nunjucks environment both places, probably via https://www.11ty.dev/docs/languages/nunjucks/#optional-use-your-nunjucks-environment